src/storage/buffer/buffer_pool.cpp

```diff
@@ -50,7 +56,8 @@ bool BufferPool::AddToEvictionQueue(shared_ptr<BlockHandle> &handle) {
 	// or the block handle is still a local variable (ConvertToPersistent)
 
 	D_ASSERT(handle->readers == 0);
-	auto ts = ++handle->eviction_timestamp;
+	auto ts = ++handle->eviction_seq_num;
+	handle->lru_timestamp_msec = GetCoarseMonotonicMillisecondTimestamp();
```

Comment:
Given that the eviction isn't enabled by default and `PurgeAgedBlocks` is never called in the common case - can we avoid making this call in all cases?

For example can we have a setting (e.g. `buffer_manager_eviction_timestamps`) - that defaults to false - and when turned off we will avoid making this call here.